crates/bdk/src/descriptor/dsl.rs

```diff
@@ -702,7 +703,7 @@ macro_rules! fragment {
         $crate::keys::make_pkh($key, &secp)
     });
     ( after ( $value:expr ) ) => ({
-        $crate::impl_leaf_opcode_value!(After, $crate::bitcoin::PackedLockTime($value)) // TODO!! https://github.com/rust-bitcoin/rust-bitcoin/issues/1302
+        $crate::impl_leaf_opcode_value!(After, $crate::bitcoin::absolute::LockTime::from_consensus($value).into()) // TODO!! https://github.com/rust-bitcoin/rust-bitcoin/issues/1302
```

Comment:
In d12242ec746ffc51da64672d7c7f7e014258d190:

You can do `$crate::miniscript::AbsLockTime::from_consensus($value)` which is much shorter than this and avoids the `.into()`. Can probably also drop the TODO at that point unless you want to file an issue on rust-miniscript suggesting that `AbsLockTIme` implement `From<u32>`.